css/layout.css

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: auto;
+  width: 480px;
+}
+
+header, aside, figure, article, footer {
+  background:#999999;
+  float: left;
+  text-align: center;
+  width: 480px;
+}
+
+header {
+  height: 75px;
+  line-height: 75px;
+}
+
+aside {
+  height: 150px;
+  line-height: 150px;
+}
+
+.cd {
+  height: 125px;
+  line-height: 125px;
+}
+
+.efgh {
+  height: 250px;
+  line-height: 250px;
+  width: 240px;
+}
+
+footer {
+  height: 80px;
+  line-height: 80px;
+}
+
+@media all and (min-width: 767px) {
+  body {
+    width: 960px;
+    margin: auto;
+  }
+  
+  header, aside, figure, article, footer {
+    background:#999999;
+    float: left;
+  }
+  
+  header {
+    height: 100px;
+    line-height: 100px;
+    width: 960px;
+  }
+  
+  aside {
+    height: 600px;
+    line-height: 600px;
+    width: 200px;
+  }
+  
+  .cd {
+    height: 200px;
+    line-height: 200px;
+    width: 380px;
+  }
+  
+  .efgh {
```

Comment:
However you want to target your second section element, followed by:
    > article {